subscriptions/subscriptions-impl/src/main/java/com/duckduckgo/subscriptions/impl/pixels/SubscriptionPixelSender.kt

```diff
@@ -194,6 +195,9 @@ class SubscriptionPixelSenderImpl @Inject constructor(
     override fun reportAddEmailSuccess() =
         fire(SUBSCRIPTION_ADD_EMAIL_SUCCESS)
 
+    override fun reportPrivacyProRedirect() =
+        fire(SUBSCRIPTION_ADD_EMAIL_SUCCESS)
```

Comment:
@marcosholgado  should be SUBSCRIPTION_PRIVACY_PRO_PATH (odd naming too) right?

Reply:
Yup, changed the name of the pixel too.